extensions/stackdriver/stackdriver.h

```diff
@@ -209,6 +209,9 @@ class StackdriverContext : public Context {
 
   // Called on onData call, so counting the data that is received.
   FilterStatus onDownstreamData(size_t size, bool) override {
+    if (!downstream_first_.has_value() && is_tcp_ &&  size > 0) {
```

Comment:
extra space in front of size
```suggestion
    if (!downstream_first_.has_value() && is_tcp_ && size > 0) {
```